src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -226,22 +237,22 @@ def _get_value(item, *args):
         return ' '
 
 
-def _get_timer_triggers(item, *args):
-    """Get all enabled timer trigger schedules from a dict.
+def _get_trigger_status(item, *args):
+    """Check if at least one enabled trigger exists in a list.
     :param dict item: The dict object
     """
     try:
         for arg in args:
             item = item[arg]
+        enabled = False
         if item:
-            schedules = []
             for trigger in item:
                 if trigger['status'].lower() == "enabled":
```

Comment:
nit: instead of using hard-coded `enabled`, you can use Enum's value like: `trigger_status.enabled.value` ?

Reply:
We do not pass a client to `_format.py` so we do not have access to the models. I prefer keeping the file more lightweight and just for json response formatting.